lib/sequenceserver.rb

```diff
@@ -57,6 +58,83 @@ def init(config = {})
 
     attr_reader :config
 
+    # 'sys' executes a shell command.
+    #
+    # 'sys' can write the stdout and/or stderr from a shell command to files, or
+    #  return these values.
+    #
+    # 'sys' can get from a failed shell command stdout, stderr, and exit status.
+    #
+    # Supply 'sys' with the shell command and optionally:
+    # A directory to change to for the duration of the execution of the shell
```

Comment:
Give the name of the corresponding keys (`directory: A directory to change to ...`).